springboard/tests/test_views.py

```diff
@@ -96,3 +98,41 @@ def test_multiple_repos(self):
             [workspace1.working_dir, workspace2.working_dir])
         self.assertEqual(indexes, views.all_pages.get_indexes())
         self.assertEqual(indexes, views.all_categories.get_indexes())
+
+    @mock.patch('unicore.google.tasks.pageview.delay')
+    def test_ga_page_titles(self, mock_task):
+        self.workspace = self.mk_workspace(name='ffl')
+        [category] = self.mk_categories(self.workspace, count=1, position=1)
+        [page] = self.mk_pages(
+            self.workspace, count=1, position=1,
+            created_at=datetime.utcnow().isoformat(),
+            primary_category=category.uuid)
+
+        app = self.mk_app(
+            self.workspace,
+            settings={'ga.profile_id': 'ID-000',
+                      'ga.persona_dimension_id': 'dimension0'})
```

Comment:
That `ga.persona_dimension_id` is specific to IoGT. Might be confusing to have it in tests here.